modules/analytics/src/backend/setup.ts

```diff
@@ -8,9 +8,9 @@ export default async (bp: typeof sdk, db: Database, interactionsToTrack: string[
 
   const removeExt = (name: string) => name?.replace(/\.flow\.json$/i, '')
 
-  process.BOTPRESS_EVENTS.on('bp_core_decision_elected', ({ channel, botId, source }) => {
+  process.BOTPRESS_EVENTS.on('bp_core_decision_elected', ({ channel, botId, source, details }) => {
     if (source === 'qna') {
-      db.incrementMetric(botId, channel, 'msg_sent_qna_count')
+      db.incrementMetric(botId, channel, 'msg_sent_qna_count', details)
```

Comment:
Why don't we get rid of bp_core_decision_elected and use bp_core_send_content instead ? It will avoid duplication. I don't see why we should keep both. It's the same metric

Reply:
Fixed in d4ea723